playbook.yaml

```diff
@@ -0,0 +1,77 @@
+- name: Install app
+  hosts: managed-nodes
+  remote_user: ec2-user
+
+  vars_prompt:
+    - name: secret_key
+      prompt: "Enter the secret key"
+      private: no
+
+    - name: trello_token
+      prompt: "Enter the Trello API Token"
+      private: no
```

Comment:
This is fine under our current model (and much easier for dev), but worth noting that on any real project I'd still recommend we switch this to `private: yes` before merging, so that we're defensively programming with awareness that in future someone might choose to run this differently where we might want extra protection to prevent leaked secrets (e.g. if running this as a pipeline job, we wouldn't want secrets in the logs)